labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -50,7 +63,7 @@ object CollectionExercise02 {
    * using a functional approach.
    */
   def groupAdultsPerAgeGroup(persons: Seq[Person]): Map[Int, Seq[Person]] = {
-    error("fix me")
+    persons filter (_.age >= 18) sortBy (_.name) groupBy (_.age / 10 * 10)
```

Comment:
No clue, I was just blindly copying the algorithm from the .java file. I had the same quesrion, but eventually (perhaps unwisely) assumed whoever implemented it wouldn't have done it if it weren't necessary